.github/workflows/coverage.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   coverage:
     name: coverage
     runs-on: ubuntu-18.04
-    timeout-minutes: 60
+    timeout-minutes: 120
```

Comment:
😢

Reply:
Not needed. I just wanted to run for longer because I am actually befuddled about why the coverage run is failing :( it passes locally, and it is failing on the envoy_mobile_main_common_test which I do not touch at all in the PR